src/App.js

```diff
@@ -10,13 +10,20 @@ class App extends Component {
     this.state = {
       test: false
     }
+    this.resetGame = this.resetGame.bind(this);
+  }
+
+  resetGame(e) {
+    e.preventDefault();
+    this.setState({ test: true })
```

Comment:
I invoke this `resetGame` function on Navbar click, because I passed this function down to Navbar as a prop called `resetGame`.